### PR TITLE
[Fix#1384]Update UserProfileUI 

### LINKED ISSUE #1384: Update the UI design of EditProfile
**Summary:**

The design of the fragment user profile looks too old and not enhancing the user interface.

**Expected behaviour:**
The UI design should look good for first-time users.

**Device **
Mi Note5 Pro 

**Screenshots:**
**The UI design right now**:-

### PR DESCRIPTION
**Pull request by iamsharad98**
Fixes #1384 

### **ScreenShots:**
WhatsApp Image 2020-03-02 at 11 44 10 PM (1)
WhatsApp Image 2020-03-02 at 11 44 10 PM

Please make sure these boxes are checked before submitting your pull request - thanks!

- [YES ] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [ YES] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [YES ] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -65,4 +65,11 @@ ext {
     cropviewVersion = '1.1.7'
     multiDexVersion = '2.0.1'
 
+    appcompatv7 = '28.0.0'
+    circleimageview = '3.1.0'
+    legacysupport = '1.0.0'
+    lifecycle = '2.1.0'
+
+
+
```
Comment (iamsh4shank) [inline build.gradle]: Remove extra lines

File: config/quality/pmd/pmd-ruleset.xml [hunk 2]
```diff
@@ -2,7 +2,8 @@
 <ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Android Application Rules"
     xmlns="http://pmd.sf.net/ruleset/1.0.0"
     xsi:noNamespaceSchemaLocation="http://pmd.sf.net/ruleset_xml_schema.xsd"
-    xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0 http://pmd.sf.net/ruleset_xml_schema.xsd">
+    xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0
+    http://pmd.sf.net/ruleset_xml_schema.xsd">
```
Comment (iamsh4shank) [inline config/quality/pmd/pmd-ruleset.xml:7]: Two lines are required ?

File: app/src/main/res/layout/snippet_top_user_profile_bar.xml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+            <include
+                layout="@layout/layout_sweet_exception_handler"
+                android:id="@+id/layout_error"
+                android:visibility="gone"/>
+
+
+        </androidx.appcompat.widget.Toolbar>
+
+
```
Comment (iamsh4shank) [inline app/src/main/res/layout/snippet_top_user_profile_bar.xml]: Remove extra lines

File: app/src/main/res/layout/snippet_top_user_profile_bar.xml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_centerVertical="true"
+                    android:text="User Details"
+                    android:textSize="20sp"
+                    android:textColor="@color/white"
+                    android:id="@+id/profileName"
+                    android:layout_marginLeft="4dp"/>
+
```
Comment (iamsh4shank) [inline app/src/main/res/layout/snippet_top_user_profile_bar.xml]: remove extra lines

File: app/src/main/res/layout/fragment_new_user_profile.xml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_below="@+id/relLayout1"
+        android:id="@+id/relLayout2">
+        <include layout="@layout/snippet_centre_user_profile"/>
+
+    </RelativeLayout>
+
```
Comment (iamsh4shank) [inline app/src/main/res/layout/fragment_new_user_profile.xml]: remove extra lines

File: app/src/main/res/layout/fragment_new_user_profile.xml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    android:layout_height="match_parent"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:id="@+id/relLayout1">
+
+        <include layout="@layout/snippet_top_user_profile_bar"/>
+
```
Comment (iamsh4shank) [inline app/src/main/res/layout/fragment_new_user_profile.xml]: remove extra lines

File: app/src/main/res/drawable/grey_border_bottom.xml [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+            <stroke
+                android:width="0.5dp"
+                android:color="@color/light_black"/>
+            <solid
+                android:color="@color/white"/>
+
+        </shape>
+    </item>
+
+
```
Comment (iamsh4shank) [inline app/src/main/res/drawable/grey_border_bottom.xml]: remove extra lines

File: app/build.gradle [hunk 8]
```diff
@@ -136,6 +136,10 @@ dependencies {
     //TableView
     implementation "com.evrencoskun.library:tableview:$rootProject.tableViewVersion"
 
+    implementation "com.android.support:appcompat-v7:$rootProject.appcompatv7"
+    implementation "de.hdodenhof:circleimageview:$rootProject.circleimageview"
+
```
Comment (iamsh4shank) [inline app/build.gradle]: remove extra lines

File: app/src/main/res/layout/snippet_top_user_profile_bar.xml [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    >
+
+    <com.google.android.material.appbar.AppBarLayout
+        android:id="@+id/app_bar_layout"
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
+        android:background="@drawable/grey_border_bottom"
+        >
+
```
Comment (iamsh4shank) [inline app/src/main/res/layout/snippet_top_user_profile_bar.xml]: remove extra lines

File: config/quality/pmd/pmd-ruleset.xml (hunk 2 above)
Comment (iamsharad98) [reply config/quality/pmd/pmd-ruleset.xml:7]: This solution is suggested by Gradle because this file has some issues and this line solves it.

**Comment by jawidMuhammadi** [comment]
Hey, @anonghostteamz Still 1 pmd violation exists in your code that is why your build is failing. As Mifos PR guidelines you have to squash and rebase all your commits in a single one. Otherwise, it will be difficult for mentors to review your code and merge it.  if you need please watch this [video](https://youtu.be/viY1BbKZhSI) on how to squash commits.

**Comment by iamsharad98** [comment]
@jawidMuhammadi Can you please tell me how to solve this pmd violation issue?

**Comment by jawidMuhammadi** [comment]
@anonghostteamz Run `./gradlew check` command in the terminal. When check finished, go to the`file:///home/travis/build/openMF/mifos-mobile/app/build/reports/pmd/pmd.html` file to see the reports. There you will find where the PMD validation is in your code.

**Pull request closed without merging**